cmd/vic-machine/create/create.go

```diff
@@ -414,6 +481,116 @@ func (c *Create) processContainerNetworks() error {
 	return nil
 }
 
+// processClientNetwork parses client network args if present
+func (c *Create) processClientNetwork() error {
```

Comment:
Aren't `processClientNetwork`, `processExternalNetwork`, and `processManagementNetwork` essentially the same but with different names? Looks like a DRY violation.